### PR TITLE
Fix a couple of typos

### PR DESCRIPTION
**Pull request by mirabilos**
• soft hyphen (U+00AD) accidentally slipped in as part of some instrument names (probably from copy/paste from Winword)
• thie → the
• while there, add a missing newline at EOF
• fix indentation (and whitespace near EOF)

### CODE REVIEW

File: mscore/data/shortcuts.xml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Shortcuts>
+ <!-- ~/.local/share/MuseScore/MuseScore3Development/shortcuts.xml -->
+ <!-- for MuseScore 3.0 (Alpha) -->
```
Comment (Jojo-Schmitz) [inline mscore/data/shortcuts.xml]: I don't think these 2 lines should be there

File: mscore/data/shortcuts.xml [hunk 2]
```diff
@@ -228,6 +230,26 @@
...
+    <seq>-</seq>
+    </SC>
+  <SC>
+    <key>add-brackets</key>
+    <seq>(</seq>
+    </SC>
+  <SC>
+    <key>no-beam</key>
+    <seq>)</seq>
+    </SC>
```
Comment (Jojo-Schmitz) [inline mscore/data/shortcuts.xml]: these are additions, not white space fixes as it says on the tin?

**Comment by anatoly-os** [comment]
Manually merged first two commits. Didn't get the purpose of the third one (added accidentally?), so please open another PR describing the purpose of the shortcuts.

**Comment by mirabilos** [comment]
Joachim Schmitz dixit:

>these are additions, not white space fixes as it says on the tin?

Oops, I accidentally hit “git commit -a” instead of just committing
what I “git add”ed. I will push a corrected branch, sorry, and thanks
for noticing!

**Pull request closed without merging**